src/Adapter/MSTest.TestAdapter/Extensions/TestCaseExtensions.cs

```diff
@@ -74,17 +75,13 @@ internal static UnitTestElement ToUnitTestElement(this TestCase testCase, string
         var isAsync = (testCase.GetPropertyValue(Constants.AsyncTestProperty) as bool?) ?? false;
         var testClassName = testCase.GetPropertyValue(Constants.TestClassNameProperty) as string;
         var name = testCase.GetTestName(testClassName);
+        var testIdGenerationStrategy = (TestIdGenerationStrategy)testCase.GetPropertyValue(
+            Constants.TestIdGenerationStrategyProperty,
+            (int)TestIdGenerationStrategy.FullyQualifiedTest);
```

Comment:
Default value is latest strategy.

Reply:
Yes! This is part of release v3, we need major bump anyways due to changes on supported TFMs and also because we did a lot of refactoring I feel more comfortable with announcing major bump as we could have break something (uncaught by our tests).

Thanks for pointing it out!